.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,24 @@
+name: Spellcheck Action
+
+on:
+  push:
+    branches:
+      - master
+      - devel
+  pull_request:
+    branches:
+      - master
+      - devel
+
+jobs:
+  build:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: rojopolis/spellcheck-github-actions@0.25.0
+      name: Spellcheck
+      with:
+        source_files: README.rst CHANGELOG.md
```

Comment:
its `CHANGELOG.rst` not `CHANGELOG.md` in the repo. This may cause some errors in spell check since its a really big file